src/fake/token_store.js

```diff
@@ -202,6 +202,7 @@ const createTokenStore = () => {
           refresh_token: generateRefreshToken(username),
           token_type: 'bearer',
           expires_in: 86400,
+          // TODO missing `scope`
```

Comment:
Something still to do?

Reply:
Not really, I just noticed this and wanted to mark it, but no need to fix it right now since there are no tests relying on this.